python/podio/test_MemberParser.py

```diff
@@ -243,6 +243,13 @@ def test_parse_valid_no_description(self):
     self.assertEqual(parsed.description, 'descriptions are not ignored even though they are not required')
     self.assertTrue(not parsed.is_builtin)
 
+  def test_parse_unit(self):
+    """Test that units are properly parsed"""
+    parser = MemberParser()
+
+    parsed = parser.parse('unsigned long long var [GeV] // description')
+    self.assertEqual(parsed.unit, 'GeV')
+
```

Comment:
Could we add another test case here with an additional default value?

Reply:
what would be a default value?

Reply:
Yes, exactly. Having a default value and a unit. Just to cover all possibilities.